core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -129,9 +129,12 @@ private[data] sealed trait OptionTInstances1 {
         fa.map(f)
     }
 
-  implicit def optionTTransLift[M[_]: Functor]: TransLift[OptionT, M] =
-    new TransLift[OptionT, M] {
-      def liftT[A](ma: M[A]): OptionT[M, A] = OptionT.liftF(ma)
+  // do NOT change this to val! I know it looks like it should work, and really I agree, but it doesn't (for... reasons)
+  implicit def optionTTransLift: TransLift.Aux[OptionT, Functor] =
```

Comment:
It just fails to resolve for some reason.  I didn't look at `-Xlog-implicits`, but it seemed like it was simply not considering the possibility.  Passing it explicitly worked fine, and obviously changing it to a def is also fine.